src/components/AppHeader.tsx

```diff
@@ -0,0 +1,23 @@
+import styles from "./AppHeader.module.css";
+import FischLogo from "../../public/fischLogo";
+import AddNewButton from "../../public/addNewButton";
+
+interface HeaderProps {
+  cardsAmt: Number;
```

Comment:
**nitpick:** Let's add more descriptive prop names instead of shortening it this way, eg: (`cardsAmount`)